src/pages/academy/components/recent-featured-posts/_style.js

```diff
@@ -42,9 +42,13 @@ export const LeftContent = styled(Flex)`
 `
 
 export const RightContent = styled(Flex)`
+    flex-direction: column;
     justify-content: start;
-    @media ${device.laptopM} {
-        flex-direction: column;
+    max-height: 464px;
+    overflow-y: scroll;
```

Comment:
Can we change this to 'auto' ? For 'scroll' the scroll bar always keeps visible to windows user.

Reply:
Yeah, we want it to be visible so that the users know to scroll it right?